src/content/reference/react-dom/server/renderToStaticNodeStream.md

```diff
@@ -29,52 +29,52 @@ const stream = renderToStaticNodeStream(<Page />);
 stream.pipe(response);
 ```
 
-[See more examples below.](#usage)
+[Buradan daha fazla örnek görebilirsiniz.](#usage)
 
-The stream will produce non-interactive HTML output of your React components.
+Akım, React bileşenlerinizin etkileşimli olmayan HTML çıktılarını üretecektir.
 
-#### Parameters {/*parameters*/}
+#### Parametreler {/*parameters*/}
 
-* `reactNode`: A React node you want to render to HTML. For example, a JSX element like `<Page />`.
+* `reactNode`: HTML'e render etmek istediğiniz bir React düğümü. Örneğin, `<Sayfa />` gibi bir JSX elemanı.
 
-#### Returns {/*returns*/}
+#### Geri Döndürür {/*returns*/}
 
-A [Node.js Readable Stream](https://nodejs.org/api/stream.html#readable-streams) that outputs an HTML string. The resulting HTML can't be hydrated on the client.
+HTML string'i döndüren bir [Okunabilir Node.js Akımı](https://nodejs.org/api/stream.html#readable-streams) döndürür. Ortaya çıkan HTML istemcide sulanamaz.
 
-#### Caveats {/*caveats*/}
+#### Uyarılar {/*caveats*/}
 
-* `renderToStaticNodeStream` output cannot be hydrated.
+* `renderToStaticNodeStream` çıktısı sulanamaz.
 
-* This method will wait for all [Suspense boundaries](/reference/react/Suspense) to complete before returning any output.
+* Bu yöntem, herhangi bir çıktı döndürmeden önce tüm [Askıya alma sınırlarının](/reference/react/Suspense) tamamlanmasını bekler.
```

Comment:
Burada Suspensi cevirmeyelim bence. React in icindeki ozel bir kavram olarak kullanilmis. Oyle kalmali. Suspense sayfasinda gerekli ceviri ve aciklama yapilacaktir zaten.